__tests__/actionsTest/editProfile/profile.test.js

```diff
@@ -0,0 +1,231 @@
+	it('should send update user profile ', async () => {
+		moxios.wait(() => {
+			const request = moxios.requests.mostRecent();
+			request.respondWith({
+				status: 200,
+				message: 'User Profile',
+				data: {
+					email: 'ricshama@gmail.com',
+					firstName: 'shema',
+					lastName: 'Eric',
+					country: 'United Arab Emirates',
+					gender: 'Male',
+					birthdate: '2003-07-18',
+					preferredlanguage: 'Swahili',
+					preferredcurrency: 'dolla ',
+					place: 'kigali',
+					department: 'software ',
+					profileImage:
+						'https://res.cloudinary.com/dby88h516/image/upload/v1581418901/barefootnomad/r6qshqeaxqdlyhmnmhtr.jpg',
+					appNotification: true,
+					emailNotification: false,
+				},
+			});
+		});
+
+		const body = {
+			email: 'ricshama@gmail.com',
+			firstName: 'shema',
+			lastName: 'Eric',
+			country: 'United Arab Emirates',
+			gender: 'Male',
+			birthdate: '2003-07-18',
+			preferredlanguage: 'Swahili',
+			preferredcurrency: 'dolla ',
+			place: 'kigali',
+			department: 'software ',
+			profileImage:
+				'https://res.cloudinary.com/dby88h516/image/upload/v1581418901/barefootnomad/r6qshqeaxqdlyhmnmhtr.jpg',
+			appNotification: true,
+			emailNotification: false,
+		};
+		const store = mockStore({});
+		await store.dispatch(updateUserProfile(body)).then(async () => {
+			expect(store.getActions()[0].type).toEqual('UPDATE_USER_PROFILE');
+		});
+	});
+
+	beforeEach(() => {
+		store.clearActions();
+	});
+});
+
+const mockedStore = configureStore(middlewares);
+const stored = mockedStore();
+describe('profile actions test ', () => {
+	beforeEach(() => {
+		moxios.install(axios);
+	});
+
+	afterEach(() => {
+		moxios.uninstall(axios);
+	});
+
+	it('CHANGE_STATE', () => {
+		const changeStateActions = [
+			{
+				type: CHANGE_STATE,
+				attribute: data,
+			},
+		];
+		stored.dispatch(changeAttribute(data));
+		expect(stored.getActions()).toEqual(changeStateActions);
+	});
+	it('CHANGE_STATE by uploading image', async () => {
+		moxios.wait(() => {
+			const request = moxios.requests.mostRecent();
+			request.respondWith({
+				status: 200,
+				response: {
+					message: 'success',
+					secure_url: 'image/image.png',
+				},
+			});
+		});
+		const data = {
+			target: {
+				files: [
+					{
+						name: 'images (2).jpeg',
+						lastModified: 1581063458236,
+						webkitRelativePath: '',
+						size: 5247,
+						type: 'image/jpeg',
+					},
+				],
+			},
+		};
+
+		const loadingImage =
+			'https://res.cloudinary.com/dby88h516/image/upload/v1580716160/barefootnomad/qv7yr2hvce9tjpx0mzh2.gif';
+
+		const expectedActions = [
+			{
+				type: HTTP_REQUEST_START,
+				attribute: { profileImage: loadingImage },
+			},
+			{
+				type: UPLOAD_IMAGE_SUCCESS,
+				attribute: {
+					profileImage: 'image/image.png',
+				},
+			},
+		];
+		const store = mockStore({});
+		await store.dispatch(uploadNewImageOnCloud(data)).then(async () => {
+			const actions = store.getActions();
+			expect(actions.length).toEqual(2);
```

Comment:
@shemaeric1234 since you have defined expectedActions, could you use them in asserting whether the actions equal to expectedActions